### DISCUSSION TITLE
BMS Support (Freedom Won + SolarMD)

### DISCUSSION THREAD

**Discussion by Cyberb29** [Louisvdw/dbus-serialbattery#465] [Ideas]
Good day, 

How can I add support for the Freedom Won + SolarMD support viA rs485, I know both these bats are supported by the kernel via Canbus, but I want to do it via RS485.

**Comment by mr-manuel** [comment]
Hello,
you can fork this repository, copy the `battery_template.py`, create and test your own battery driver and the add a PR to merge it.
